### ISSUE TITLE
This app isn't verified

### ISSUE THREAD

**Issue by pirofti** [dsoprea/GDriveFS#211]
Hi,

I managed to install GDriveFS on OpenBSD, but now during authentication I get a message saying ["This app isn't verified"](https://lh3.googleusercontent.com/DZ9tBXY0K4fDxWe0K-AKkX8j1drhDlCEmvnSAOKoSPaQomg-sqJa9YM6MmFlVgVlEZY=w500) from Google.

Is there anything I can do on my end to help the process? Do you need to click a few things on your side to verify the app? Or is this all up to Google now?

Thank you,
Paul Irofti

**Comment by dsoprea** [comment]
Do you mean in the OAuth consent screen? Yeah. I recently got screwed by Google. The keys I had were suddenly dropped out of their system, and the new system requires a lot more steps and formal approvals. This fell off my radar. I'll try to get it done this week.

**Comment by pirofti** [comment]
Thank you for looking into this!

**Comment by dsoprea** [comment]
Resubmitted for approval. They're being a pain.

**Comment by dsoprea** [comment]
Still working on it.

**Comment by pirofti** [comment]
Thank you!

**Comment by dsoprea** [comment]
Continued back-and-forth. Was required to give them a video showing usage.

**Comment by beejjorgensen** [comment]
Jeez. I admire your tenacity. Looking forward to this working again.

**Comment by dsoprea** [comment]
Since I myself am locked out, I'm also limited in what I can give them.



Dustin

**Comment by Zekromaster** [comment]
Any updates about this?

**Comment by dsoprea** [comment]
They've given me some supposedly experimental credentials so that I can get past the user-cap issue that was preventing me from accessing the account. However, I've submitted several videos to them and they keep asking for adjustments in what I'm presenting. Currently, I'm blocked on not being able to write data to my Drive using those credentials, though I can create directories seemingly without a problem.

**Comment by dsoprea** [comment]
Sorry for the delay.

**Comment by dsoprea** [comment]
The API changed while we've been dealing with this. I've pushed a fix and posted another explanatory video to them.

**Comment by pirofti** [comment]
This might work in the meantime? I tried it with Grive2 and it did.

https://github.com/vitalif/grive2#different-oauth2-client-to-workaround-over-quota-and-google-approval-issues

**Comment by dsoprea** [comment]
You can also generate your own API credentials using the two scopes we require:

https://github.com/dsoprea/GDriveFS/blob/90375da42d1a1cb6241ae44b151f46761b7adebf/gdrivefs/oauth_authorize.py#L51

I've pushed a branch with an alternative set of OAuth credentials for testing things that I am not using:

https://github.com/dsoprea/GDriveFS/tree/dustin/temp_credentials

**Comment by dsoprea** [comment]
This should now be resolved. Please confirm that it now works for you.

**Comment by dsoprea** [comment]
Bump? Please test.

**Comment by pirofti** [comment]
I confirm that this works now with what is in the master branch. Thank you very much for having the interest and patience of taking care of this!

gdfs now fails to mount, but I think that is because of versioning issues. I will open a separate issue if I can not figure it out by myself.

**Comment by dsoprea** [comment]
Let me know if I can help. You can mount in the foreground by doing the
mount with gdfstool and setting GD_DEBUG=1 in the environment.
